chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -156,6 +156,11 @@ BraveRenderViewContextMenu::BraveRenderViewContextMenu(
 
 bool BraveRenderViewContextMenu::IsCommandIdEnabled(int id) const {
   switch (id) {
+    case IDC_CONTENT_CONTEXT_FORCE_PASTE:
+      // only enable if there is plain text data to paste - this is what
+      // IsPasteAndMatchStyleEnabled checks internally, but IsPasteEnabled
+      // allows non text types
+      return IsPasteAndMatchStyleEnabled();
```

Comment:
it seems we don't actually have tests for this class, but they could be useful

Reply:
yes, I've been waiting to merge this because I need to add tests

Reply:
seems there are no upstream paste tests so I'm going to leave this for now and maybe do a follow-up